backend/clubs/models.py

```diff
@@ -965,6 +965,15 @@ def __str__(self):
         )
 
 
+class Search(models.Model):
+    person = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+    query = models.CharField(max_length=255)
```

Comment:
You will receive an error if the user passes in something greater than 255 characters. Either switch to a `TextField`, or truncate the input if it is over 255 characters.

Reply:
Yeah I was thinking that no one would enter a search that long, but pays to be safe!